packages/@aws-cdk/aws-applicationsignals-alpha/lib/index.ts

```diff
@@ -4,3 +4,5 @@ export * from './enablement/constants';
 export * from './enablement/instrumentation-versions';
 export * from './enablement/ecs-cloudwatch-agent';
 export * from './enablement/ecs-sdk-instrumentation';
+export * from './slo/slo';
+export * from './slo/slo-types';
```

Comment:
Add an index.ts in the slo dir, and expose whatever you want there. Here, add only the directory.